tests/testthat/test-standardization.R

```diff
@@ -0,0 +1,15 @@
+test_that("standardized risk ratio", {
+  dt11 = wcgs %>%
+    group_by(dibpat) %>%
+    count(smoke, chd69) %>%
+    mutate(pL = sum(n) / total) %>%
```

Comment:
Where did you get this `total` column from? I think there are some mistakes her.